content/blog/how-to-provision-hpe-greenlake-private-cloud-resources-from-servicenow-service-catalog.md

```diff
@@ -0,0 +1,175 @@
+---
+title: How to provision HPE GreenLake private cloud resources from ServiceNow
+  Service Catalog
+date: 2022-03-08T06:13:00.000Z
+author: Vinnarasu Ganesan, Thavamaniraja S
+authorimage: /img/Avatar1.svg
+thumbnailimage: /img/thumbnail.jpg
+tags:
+  - hpe-greenlake
+---
+## Introduction
+
+A key strength of HPE GreenLake for private cloud is self-service
+orchestration and automation, which you do from the HPE GreenLake for
+private cloud dashboard and can be consumed using API and IaC. However,
+some ServiceNow users prefer to provision resources from the ServiceNow
+Service Catalog. For these users, HPE GreenLake for private cloud offers
+a free Morpheus plugin for ServiceNow, which they can install from the
+ServiceNow Store. After you install and configure the plugin, HPE
```

Comment:
Once the plugin installed and configured, HPE GreenLake for private cloud catalog items (In the previous blog, we discussed how to curate and expose service catalog items using HPE GreenLake for private cloud) can be presented in the ServiceNow Service Catalog for ordering.